src/components/shared/Draggable.js

```diff
@@ -92,6 +92,8 @@ export class Draggable extends React.PureComponent<Props, State> {
     mouseMoveHandler: MouseEvent => void,
     mouseUpHandler: MouseEvent => void
   ) {
+    this._uninstallMoveAndUpHandlers();
```

Comment:
Can you please add a comment explaining the various cases where this is needed?

Reply:
done